unstructured/partition/text.py

```diff
@@ -206,12 +208,56 @@ def is_empty_bullet(text: str) -> bool:
     return UNICODE_BULLETS_RE.match(text) and len(text) == 1
 
 
+def is_in_header_position(
+    text: str,
+    coordinates: Optional[Tuple[Tuple[float, float], ...]] = None,
+    coordinate_system: Optional[CoordinateSystem] = None,
+    threshold: float = 0.05,
+) -> bool:
+    """Checks to see if the position of the text indicates that the text belongs
+    to a header."""
+    if coordinates is None or coordinate_system is None:
+        return False
+
+    min_y = min([coordinate[1] for coordinate in coordinates])
```

Comment:
Am I reading this right that it only checks the y-coordinate of the upper left corner of the bbox? Seems like if the bbox is really tall and only the top edge is in the zone, we could have a false positive.

Reply:
Hah! Totally forgot about this PR. Yeah it's just checking the top/bottom corner right now. What about checking the mid-point between the y coordinates instead?